.github/workflows/deploy.yml

```
@@ -0,0 +1,258 @@
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build packages
+        run: npm run build:packages
+
+      - name: Build web application
+        run: NODE_OPTIONS=--max-old-space-size=8192 npm run build --workspace=@schwalbe/web
+
+      - name: Deploy rollback to Production
+        uses: amondnet/vercel-action@v25
+        if: secrets.VERCEL_TOKEN != '' && secrets.VERCEL_ORG_ID != '' && secrets.VERCEL_PROJECT_ID != ''
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)